ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -13137,13 +13136,12 @@ private void updateDefaultTblProps(Map<String, String> source, Map<String, Strin
    *
    * @param tblProp
    *          property map
-   * @param isCTAS
```

Comment:
I've manually tested the patch; when the translator is enabled; running
```
create table b as select 1 aa
```
results in 
```
Caused by: org.apache.hadoop.hive.metastore.api.MetaException: An external table's location should not be located within managed warehouse root directory, table:b,location:/data/hive/warehouse/b,managed warehou
se:/data/hive/warehouse
        at org.apache.hadoop.hive.metastore.MetastoreDefaultTransformer.validateTablePaths(MetastoreDefaultTransformer.java:932) ~[hive-exec-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
```

running the same command with translator disabled results in an EXTERNAL table which is also bad...